static/_redirects

```diff
@@ -79,6 +79,7 @@
 /docs/concepts/jobs/run-to-completion-finite-workloads/      /docs/concepts/workloads/controllers/job/ 301
 /id/docs/concepts/jobs/run-to-completion-finite-workloads/      /id/docs/concepts/workloads/controllers/job/ 301
 /docs/concepts/nodes/node/     /docs/concepts/architecture/nodes/ 301
+/docs/storage-force-detach-on-timeout/ /docs/concepts/architecture/nodes/#storage-force-detach-on-timeout 301
```

Comment:
Why are we adding this redirection?

Reply:
This redirection was pretty much the point of this change.

I was asked to document the point of the new KCM flag that we added here https://github.com/kubernetes/kubernetes/pull/120344.

We added a number of details to the CLI argument as a part of the code review process, but the feedback was that too much was added to the CLI.

It was suggested to direct the CLI to just point to the website to explain the technical implications of setting this flag. So the CLI now points to this exact redirect; this has been changed in kubernetes 1.30.

CCing @liggitt @carlory @msau42 who have some context here.

Reply:
See [here](https://github.com/kubernetes/kubernetes/pull/120344#discussion_r1333811968) for the relevant context.

Reply:
Thanks.
- This should be a 302 redirect, not 301; please change it here
- I'd have assigned https://k8s.io/node-storage-force-detach-on-timeout as the URL (no `/docs`)
  (best to check with SIG Docs before minting a short URL)

Reply:
I don't feel strongly either way... if we can get the flag doc merged asap, shortening this seems fine to me

Reply:
(but, definitely a **302** redirect so we're not saying it's always a redirect to that specific page)

Reply:
I'm uploading the 302 redirect now; thanks.